src/basho_bench_duration.erl

```diff
@@ -165,3 +176,25 @@ worker_stopping(WorkerPid) ->
     %% WorkerPid is basho_bench_worker's id, not the pid of actual driver 
     gen_server:cast(?MODULE, {worker_stopping, WorkerPid}),
     ok.
+
+maybe_add_rampup(State) ->
+    case basho_bench_config:get(workers_rampup, undefined) of
+        undefined ->
+            State;
+        Interval when is_integer(Interval) ->
+            State#state{rampup_interval=Interval};
+        %% TODO: should we support per type intervals?
```

Comment:
Maybe open a small ticket to document the idea for later consideration.

Reply:
Will do.